cst438_gradebook/src/main/java/com/cst438/controllers/GradeBookController.java

```diff
@@ -172,7 +172,7 @@ private Assignment checkAssignment(int assignmentId, String email) {
 	@Transactional
 	public void addAssignment (@PathVariable("name") String name, @PathVariable("date") String date, 
 			@PathVariable("id") int courseId) {
-		
+		int x = 0;
```

Comment:
I see you added this since you originally pushed changes on dev to main. I reviewed the code you pushed to main and everything looks good. Approving PR.